  <launch>
  <node name="OSListener" pkg="quadcopter_ctrl" type="OnlineSearchListener" args="1" output="screen"/>
  <node name="quadPatrolGRAPH_0" pkg="quadcopter_ctrl" type="quadPatrolGRAPH" args="0 $(arg input) 5.0 sim 0 1 0" output="screen"/>
<!--  <node name="quadPatrolGRAPH_1" pkg="quadcopter_ctrl" type="quadPatrolGRAPH" args="1 $(arg input) 5.75 sim 0 1" output="screen"/>
  <node name="quadPatrolGRAPH_2" pkg="quadcopter_ctrl" type="quadPatrolGRAPH" args="2 $(arg input) 6.5 sim 0" output="screen"/>
  <node name="quadPatrolGRAPH_3" pkg="quadcopter_ctrl" type="quadPatrolGRAPH" args="3 $(arg input)" output="screen"/>
  <node name="quadPatrolGRAPH_4" pkg="quadcopter_ctrl" type="quadPatrolGRAPH" args="4 $(arg input)" output="screen"/>
  <node name="quadPatrolGRAPH_5" pkg="quadcopter_ctrl" type="quadPatrolGRAPH" args="5 $(arg input)" output="screen"/>
  <node name="quadPatrolGRAPH_6" pkg="quadcopter_ctrl" type="quadPatrolGRAPH" args="6 $(arg input)" output="screen"/>
  <node name="quadPatrolGRAPH_7" pkg="quadcopter_ctrl" type="quadPatrolGRAPH" args="7 $(arg input)" output="screen"/>
  <node name="quadPatrolGRAPH_8" pkg="quadcopter_ctrl" type="quadPatrolGRAPH" args="8 $(arg input)" output="screen"/>
  <node name="quadPatrolGRAPH_9" pkg="quadcopter_ctrl" type="quadPatrolGRAPH" args="9 $(arg input)" output="screen"/>
-->
</launch>
